raiden/console.py

```diff
@@ -228,8 +227,8 @@ def ping(self, peer):
             print("Error: peer {} not found in discovery".format(peer))
             return
 
-        nonce = self._ping_nonces[peer]
-        self._ping_nonces[peer] += 1
+        nonce = self._raiden.protocol.ping_nonces[peer]
+        self._raiden.protocol.ping_nonces[peer] += 1
         msg = Ping(nonce)
         self._raiden.sign(msg)
         return self._raiden.protocol.send_and_wait(peer.decode('hex'), msg)
```

Comment:
Btw, using `send_and_wait` adds the `Ping` message into the queue, this was probably unintentional.